frontend/public/locales/en/public.json

```diff
@@ -136,8 +136,18 @@
   "Completed": "Completed",
   "Release notes": "Release notes",
   "Cluster Settings": "Cluster Settings",
-  "Global configuration": "Global configuration",
+  "Configuration": "Configuration",
   "ClusterVersion details": "ClusterVersion details",
+  "{{kind}} details": "{{kind}} details",
+  "{{label}}": "{{label}}",
```

Comment:
What is the context for this message? Usually if the entire message is a variable it doesn't make sense to externalize it.

Reply:
This is [an existing entry](https://github.com/openshift/console/blob/85086217feb35e64c36d68a8d315fe59275e510b/frontend/public/locales/en/public.json#L160) that is just getting reordered.